scripts/models/Image.js

```diff
@@ -0,0 +1,24 @@
+import { Media } from './Media.js';
+
+export class Image extends Media {
+  constructor(data) {
+    super(data);
+    if (data.image !== undefined) {
+      this._image = data.image;
+    } else {
+      this._image = '';
+    }
+    if (data.title !== undefined) {
+      this._title = data.title;
+    } else {
+      this._title = '';
+    }
+  }
+
+  get mediaHtml() {
+    return `<img src="${this._image}" alt="${this._title}"></img>`;
+  }
+  get path() {
+    return this._image || '';
```

Comment:
`this._image` is always defined as it falls back to an empty string in the constructor, so no need to do it again